scheduler/controllers/collectorgroups/datacollection.go

```diff
@@ -2,23 +2,37 @@ package collectorgroups
 
 import (
 	odigosv1 "github.com/odigos-io/odigos/api/odigos/v1alpha1"
+	"github.com/odigos-io/odigos/common"
 	"github.com/odigos-io/odigos/k8sutils/pkg/consts"
 	"github.com/odigos-io/odigos/k8sutils/pkg/env"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func NewNodeCollectorGroup() *odigosv1.CollectorsGroup {
+const NodeCollectorDefaultOwnMetricsPort = int32(55682)
```

Comment:
see another comment about moving this const to common.
Also, this default value should be defined in helm, right?